### PR TITLE
IntesisHome Integration Docs: Fixed broken URLs

### PR DESCRIPTION
**Pull request by erlendmoland**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/intesishome.markdown [hunk 1]
```diff
@@ -10,7 +10,7 @@ ha_codeowners:
 ha_domain: intesishome
 ---
 
-The `IntesisHome` climate platform lets you control [IntesisHome](https://www.intesishome.com) and [Airconwithme](https://www.airconwithme.com) devices. IntesisHome provides integrations with air conditioners, including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.
+The `IntesisHome` climate platform lets you control [IntesisHome](https://www.intesishome.com/) and [Airconwithme](https://airconwithme.com/) devices. IntesisHome provides integrations with air conditioners, including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.
```
Comment (klaasnicolaas) [inline source/_integrations/intesishome.markdown]:
```suggestion
The `IntesisHome` climate platform lets you control [IntesisHome](https://www.intesishome.com) and [Airconwithme](https://airconwithme.com) devices. IntesisHome provides integrations with air conditioners, including Panasonic, Daikin, Fujitsu, Toshiba, LG and more.
```

File: source/_integrations/intesishome.markdown [hunk 2]
```diff
@@ -27,7 +27,7 @@ climate:
 
 {% configuration %}
 username:
-  description: "Your username for [IntesisHome.com](https://user.intesishome.com) / [Airconwithme](https://user.airconwithme.com)"
+  description: "Your username for [IntesisHome.com](https://accloud.intesis.com/login) / [Airconwithme](https://airconwithme.com)"
```
Comment (klaasnicolaas) [inline source/_integrations/intesishome.markdown]:
```suggestion
  description: "Your username for [IntesisHome.com](https://accloud.intesis.com) / [Airconwithme](https://airconwithme.com)"
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/intesishome.markdown (hunk 1 above)
Comment (erlendmoland) [reply source/_integrations/intesishome.markdown]: @klaasnicolaas Ah, missed that one. Thanks for verifying!

**Review by frenck — APPROVED** [review]
Thanks, @erlendmoland 👍 

Happy Hacktoberfest 🎉

**Pull request merged**